### ISSUE TITLE
Accessibility - City - VoiceOver Not Reading Link on "Industry - Sectors" Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Local-Labor-Market-Data-IOS#18]
While reading the "Industry - Sectors" screen for"Total Private" for Akron, OH, I noticed the link "Total Nonfarm" which VoiceOver does not read as a link. If I tap on the link, it opens but doesn't tell the person it's a link.

Please see attached screenshot.

Highlighted "Total Nonfarm" link

![bls local data - ipad - city screen - highlighting total nonfarm link](https://user-images.githubusercontent.com/10423487/56597779-30015d00-65c1-11e9-8368-51c5708fd18b.png)

**Comment by chawlani** [comment]
This label has list of all the parents and each word is clickable. 
Ex - For Goods producing the parents list is Total Nonfarm->Total Private where each word (Total Nonfarm and Total Private) is clickable and they are not separate buttons. If you would like, I can hide this label and voice over is running

**Comment by jfc3-dol** [comment]
No that fine to have it there I need VoiceOver to know and read it as a link or button is all if people can use it to go somewhere in the application. Not sure which is best to be but if I had to take a quick guess I would go with a link because it goes somewhere and is not a function.

**Comment by chawlani** [comment]
And I said, its a one label but with ability for each word to be clickable to go to different screen, which is not possible to do with accessibility
Ex - " Total Nonfarm->Total Private" is one label but is attributed so that Total Nofarm and Total Private are both clickable. But for accessibility, they can't be reading it as two items.
I can definitely make the label as link

**Comment by jfc3-dol** [comment]
At least being read as a link the VoiceOver user would know it goes somewhere instead of assuming it's plain text.

**Comment by jfc3-dol** [comment]
VoiceOver is now reading the text as a link but when there is more than one link/item it's being read as one link instead of two or more. 

Please see attached screenshot.

![bls - ios - detail screen links should be more than one](https://user-images.githubusercontent.com/10423487/56980636-da8df880-6b4a-11e9-999b-d1f70dc297ff.png)

**Comment by jfc3-dol** [comment]
I can tap on the individual links when VoiceOver is off and it takes me to the correct screen but when VoiceOver is on it only goes back one screen because it's acting as one link.

**Comment by chawlani** [comment]
When voiceover is running, We can display only parent link instead of list of parent and ancestors. There are other ways to navigate parents and grandparents.

**Comment by jfc3-dol** [comment]
Think the way you came up with for VoiceOver users only seeing the previous levels in the bread crumbs as non-links is great but when not using VoiceOver they are links. Only thing I noticed different is that the non-VoiceOver set of bread crumbs had the current page the user was on and the VoiceOver one did not or am I incorrect in what you had updated/created.

If that is the way it's supposed to work I will close the issue.

**Comment by jfc3-dol** [comment]
After we talked and I looked at this again the bread crumbs were the same as they were, but they were not links when using VoiceOver.

Please see the two screenshots showing the difference.

**Detail screen with bread crumbs as links.**

![bls - ios - detail screen - bread crumbs are links](https://user-images.githubusercontent.com/10423487/57246051-559f5500-700a-11e9-9063-29308c1ec856.png)

**Detail screen with bread crumbs without links because VoiceOver is on.**

![bls - ios - detail screen - bread crumbs have no links using voiceover](https://user-images.githubusercontent.com/10423487/57246061-5b953600-700a-11e9-8bec-66c8fd292424.png)

Going to close issues because the screen is acting as expected/designed.
